clio-client/src/main/scala/org/broadinstitute/clio/client/commands/Commands.scala

```diff
@@ -1,40 +1,30 @@
 package org.broadinstitute.clio.client.commands
 
 import akka.http.scaladsl.model.HttpResponse
-import com.typesafe.scalalogging.LazyLogging
-import org.broadinstitute.clio.client.ClioClientConfig
+import enumeratum._
+import org.broadinstitute.clio.client.parser.BaseArgs
+import org.broadinstitute.clio.client.webclient.ClioWebClient
 
-import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.collection.immutable.IndexedSeq
+import scala.concurrent.{ExecutionContext, Future}
+
+sealed trait CommandType extends EnumEntry
+
+object Commands extends Enum[CommandType] {
+  override val values: IndexedSeq[CommandType] = findValues
+
+  case object AddWgsUbam extends CommandType
+  case object QueryWgsUbam extends CommandType
```

Comment:
Do we care that these will be PascalCase instead of camelCase on the command line?